common/src/main/java/net/william278/huskhomes/redis/redisdata/RedisPubSub.java

```diff
@@ -0,0 +1,58 @@
+package net.william278.huskhomes.redis.redisdata;
+
+import io.lettuce.core.pubsub.RedisPubSubListener;
+
+public abstract class RedisPubSub<K,V> implements RedisPubSubListener<K,V> {
```

Comment:
Does lettuce not provide their own base impl of this we can extend or instantiate wherever we need this? Kind of annoying to have to have a skeleton class like this in our codebase

Reply:
Yes, but you should implement all the 6 methods every time. This could reduce readability.

Reply:
> Yes, but you should implement all the 6 methods every time. This could reduce readability.

We're not implementing these methods, though -- we're leaving them with an empty body. I think this harms readability in fact as we're introducing extra needless code into the project